nuget.config

```diff
@@ -4,4 +4,7 @@
         <add key="enabled" value="True" />
         <add key="automatic" value="True" />
     </packageRestore>
+    <packageSources>

```

Comment:
I don't understand nuget 😩   Other documentation told me to add it to the csproj, which I went and did in https://github.com/nhydock/BepInEx.UnityInput/commit/da2c4c5465d342bf7061893df0bf15bfe8c19b31

https://docs.microsoft.com/en-us/nuget/reference/msbuild-targets?WT.mc_id=DT-MVP-5003978#restore-properties

Reply:
Fair enough, I missed that you added it in the meantime. This one's pointless then.